### PR TITLE
#24 Bump Kotlin and Compose

### PR DESCRIPTION
**Pull request by BorysChajdas**

### CODE REVIEW

**Review by Nataniel-Antosik — DISMISSED** [review]

**Review by marcinWisnia — DISMISSED** [review]

**Review by bartek977 — APPROVED** [review]

**Review by Nataniel-Antosik — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -48,7 +48,7 @@ android {
 
 dependencies {
 
-    def composeBom = platform('androidx.compose:compose-bom:2022.12.00')
+    def composeBom = platform('androidx.compose:compose-bom:2023.01.00')
     implementation composeBom
     androidTestImplementation composeBom
```
Comment (MarekMacko) [inline app/build.gradle:53]: Compose it's not used in instrumentation tests and it should be removed

File: build.gradle [hunk 2]
```diff
@@ -1,5 +1,5 @@
 plugins {
     id 'com.android.application' version '7.4.0' apply false
     id 'com.android.library' version '7.4.0' apply false
```
Comment (MarekMacko) [inline build.gradle:3]: Doesn't seem be required, we don't have any submodules, only the main "app"

File: app/build.gradle (hunk 1 above)
Comment (BorysChajdas) [reply app/build.gradle:53]: Sure, I can handle it in next PR

File: build.gradle (hunk 2 above)
Comment (BorysChajdas) [reply build.gradle:3]: I can handle it in next PR

**Pull request merged**